tests/system/Commands/Database/ShowTableInfoTest.php

```diff
@@ -0,0 +1,152 @@
+
+        $result = $this->getResultWithoutControlCode();
+
+        $expected = 'List of Metadata Information in Table "db_migrations":';
+        $this->assertStringContainsString($expected, $result);
+
+        $result   = preg_replace('/\s+/', ' ', $result);
+        $expected = <<<'EOL'
+            | Field Name | Type | Max Length | Nullable | Default | Primary Key |
+            EOL;
+        $this->assertStringContainsString($expected, $result);
+    }
+
+    public function testDbTableDesc(): void
+    {
+        $seeder = Database::seeder();
+        $seeder->call(CITestSeeder::class);
+
+        command('db:table db_user --desc');
+
+        $result = $this->getResultWithoutControlCode();
+
+        $expected = 'Data of Table "db_user":';
+        $this->assertStringContainsString($expected, $result);
+
+        $expected = <<<'EOL'
+            +----+--------------------+--------------------+---------+------------+------------+------------+
+            | id | name               | email              | country | created_at | updated_at | deleted_at |
+            +----+--------------------+--------------------+---------+------------+------------+------------+
+            | 4  | Chris Martin       | chris@world.com    | UK      |            |            |            |
+            | 3  | Richard A Cause... | richard@world.c... | US      |            |            |            |
+            | 2  | Ahmadinejad        | ahmadinejad@wor... | Iran    |            |            |            |
+            | 1  | Derek Jones        | derek@world.com    | US      |            |            |            |
+            +----+--------------------+--------------------+---------+------------+------------+------------+
+            EOL;
+        $this->assertStringContainsString($expected, $result);
+    }
+
+    public function testDbTableLimitFieldValueLength(): void
+    {
+        command('db:table db_user --limit-field-value 5');
+
+        $result = $this->getResultWithoutControlCode();
+
+        $expected = 'Data of Table "db_user":';
+        $this->assertStringContainsString($expected, $result);
+
+        $expected = <<<'EOL'
+            +----+----------+----------+---------+------------+------------+------------+
+            | id | name     | email    | country | created_at | updated_at | deleted_at |
+            +----+----------+----------+---------+------------+------------+------------+
+            | 1  | Derek... | derek... | US      |            |            |            |
+            | 2  | Ahmad... | ahmad... | Iran    |            |            |            |
+            | 3  | Richa... | richa... | US      |            |            |            |
+            | 4  | Chris... | chris... | UK      |            |            |            |
+            +----+----------+----------+---------+------------+------------+------------+
+            EOL;
+        $this->assertStringContainsString($expected, $result);
+    }
```

Comment:
It may be useful
```
    public function testDbTableLimitRows(): void
    {
        command('db:table db_user --limit-rows 2');

        $result = $this->getResultWithoutControlCode();

        $expected = 'Data of Table "db_user":';
        $this->assertStringContainsString($expected, $result);

        $expected = <<<'EOL'
            +----+--------------------+--------------------+---------+------------+------------+------------+
            | id | name               | email              | country | created_at | updated_at | deleted_at |
            +----+--------------------+--------------------+---------+------------+------------+------------+
            | 1  | Derek Jones        | derek@world.com    | US      |            |            |            |
            | 2  | Ahmadinejad        | ahmadinejad@wor... | Iran    |            |            |            |
            +----+--------------------+--------------------+---------+------------+------------+------------+
            EOL;
        $this->assertStringContainsString($expected, $result);
    }
    public function testDbTableAllOptions(): void
    {
        command('db:table db_user --limit-rows 2 --limit-field-value 5 --desc');

        $result = $this->getResultWithoutControlCode();

        $expected = 'Data of Table "db_user":';
        $this->assertStringContainsString($expected, $result);

        $expected = <<<'EOL'
            +----+----------+----------+---------+------------+------------+------------+
            | id | name     | email    | country | created_at | updated_at | deleted_at |
            +----+----------+----------+---------+------------+------------+------------+
            | 4  | Chris... | chris... | UK      |            |            |            |
            | 3  | Richa... | richa... | US      |            |            |            |
            +----+----------+----------+---------+------------+------------+------------+
            EOL;
        $this->assertStringContainsString($expected, $result);
    }
```

Reply:
Thanks! And I found the sorting is not so easy and a bug. 
Because we need a sort key (column), but finding the right sort key is not easy.

Reply:
LGTM! 💯